### ISSUE TITLE
Branding

### ISSUE THREAD

**Issue by effgarces** [LibreBooking/app#7]
While there hasn't been much development, besides solving some bugs and a first maintenance release, it seems logical that the branding for this project should change.
I'm considering creating a new project based on booked, with a new name and image, all development would then occur under that new project, hopefully avoiding any kind of potential legal problem.
I'm open to sugestions, wether names, logos, etc, so feel free to share any ideas.

**Comment by treborin** [comment]
Hi there, there is a latin word i like for this as a new name: LIBRI. Hope you like it.
Btw, thanks to all the people who try to continue this great  project. I'll try my best to get time to help in some way.
Good luck :)

**Comment by effgarces** [comment]
Hi, sorry it took me so long to answer. I like it, it seems fitting, but I don't know if it will be adequate, there's already Libreoffice out there, and I also found a couple of instances of this name being used for software. I need to check it further, to ensure that there's no potential problems.
I will be adding other sugestions in a nearby future and hopefully choosing something with some comunity feedback.

**Comment by dsoundmn** [comment]
I've set up a fork in my org with a new name and a free open source forever promise. if anyone is interested in pulls, bugs, and features - let's keep this alive and make it shine - https://github.com/americandigitalservices/KairosCalendar

**Comment by lourdas** [comment]
It's very unfortunate to read that the original author decided to abandon maintenance of the project.

Regarding the name, what about **LibriCal**? I think it is somewhat euphonic.

In the sourceforge repository, I had created a ticket for the Greek translation almost 3 years ago, which was never committed. I will update the translation against the 2.8.x version and open a pull request here.

**Comment by effgarces** [comment]
> I've set up a fork in my org with a new name and a free open source forever promise. if anyone is interested in pulls, bugs, and features - let's keep this alive and make it shine - https://github.com/americandigitalservices/KairosCalendar

Awesome, the more the merrier, I'm hoping that all of the interest will eventually to a main version that the community version can rally to. In the meantime I will be adding mainly fixes to my repo and an occasional feature, so feel free to grab whatever you like or that is usefull.

**Comment by effgarces** [comment]
> It's very unfortunate to read that the original author decided to abandon maintenance of the project.
> 
> Regarding the name, what about **LibriCal**? I think it is somewhat euphonic.
> 
> In the sourceforge repository, I had created a ticket for the Greek translation almost 3 years ago, which was never committed. I will update the translation against the 2.8.x version and open a pull request here.

True, but I'm hoping the community will rally around one of the forks, so maybe in a near future we will have a regularly maintained version.

Well the name is similar to the previous sugestion, but the interesting bit is when you search for it in googlle,  check the urbandictionary link.

I just added your pull request to the features branch, after confirmation that all seems to in working order, I will move it to the RC branch and after awhile finally to Final, at that point I usually do a maintenance release.

**Comment by colisee** [comment]
Hi, 
I am running Booked v2.8.4 and I am glad I found your repository. 

Regarding the new name, what about Librebooking since you referred to Libreoffice? An alternative could be Openbooking...

**Comment by YZone** [comment]
Hi,
Thanks for keeping this project alive.
About branding: we all want a piece of the available resources, call it **Claimed!**

**Comment by gparami** [comment]
I see most of us here calling it "Booked". Can't we just call it "Booked"?

**Comment by effgarces** [comment]
The problem is that I/we don't have the rights to that name, it belongs to TwinkleToes Software/Nick. While we don't pose any threat, software history is full of frivolous lawsuits some with disastrous consequences.
As the original Booked Scheduler keeps moving on, this last open source release keeps getting more and more distant so invariably the two will keep getting confused that is something that I'm keen to avoid.

Thanks everyone for their suggestions and I'm sorry for taking so long with this, unfortunately due the pandemic and some work related changes, the little bit of free time that I had to do something here as been further reduced, but as soon as I clear some of the tasks currently pending, I'm hoping to  get back into booked and do a couple more maintenance releases.

**Comment by lourdas** [comment]
You are right about the name, it must change. I suggest you gather all name options and create a poll or something so that we can all vote.

**Comment by effgarces** [comment]
Thanks for the suggestion, that is probably the  best route to take, I wil leave this issue open for a few more weeks, in case someone has some more sugestions and afterwards, I will create a poll and share it here, so that everyone can weigh in on this matter.

**Comment by effgarces** [comment]
By the way I will also leave my suggestion.  Requisita-me
It's portuguese for "book me", in fact we are using a condensed form of it Requisit@M

P.S. While we are using this for our implementation, we don't have any claims/trademarks to the name, so if this one gets chosen there shouldn't exist any issues with it.

**Comment by simonflood** [comment]
Having started using this fork of Booked Scheduler and found this thread I've been having a think and search to find a name which could work and doesn't appear to already be used. So far I've come up with just one word:

calendrical

It means relating to or characteristic of or used in a calendar or time measurement.

**Comment by stevleibelt** [comment]
I really like the name "Librebooking". It shows the flavor, I can't find any software claimed that name, it shows what kind of problem the software wants to solve plus it is open enough so the software doesn't care if you manage horses or ships.

**Comment by effgarces** [comment]
Thanks everyone for you suggestions. I think that more than enough time has passed, so I will be gathering all of the names on a poll that I will post here, during the next month, I will let it run for at least 1 month, so that everyone interested  have their say, maybe more depending on my work schedule. 
I hope to finally close this issue during July with a full renaming of the project.

**Comment by effgarces** [comment]
As promised, I've made a simple poll in google forms with all suggestions. You can add your vote here: https://forms.gle/KzPfdR17p7t8Ktws9 
Please take into account the issues mentioned with certain names.

After letting this poll run for a month or two I will share the results here, after that, I'm hoping we will have a clear winner, if not I / we should carefully analyse the top results and make a decision based on the community feedback.

**Comment by effgarces** [comment]
![image](https://user-images.githubusercontent.com/1185683/122678269-f5fe6680-d1dd-11eb-9ff9-01792428df77.png)

**Comment by effgarces** [comment]
Thanks for the input, it seems that the way to go is "Librebooking". In the following weeks/months I will be changing the branding to it with the contribution of a friend that will be handling the design work.

**Comment by JohnVillalovos** [comment]
Seems like the branding work has been completed.
